language/diem-vm/src/system_module_names.rs

```diff
@@ -39,13 +39,13 @@ pub static CHECK_UPGRADE: Lazy<Identifier> =
 static UPGRADE_MODULE_NAME: Lazy<Identifier> =
     Lazy::new(|| Identifier::new("Upgrade").unwrap());
 
-// TODO: this will be deprecated in 5.0.11
-pub static RESET_PAYLOAD: Lazy<Identifier> =
-    Lazy::new(|| Identifier::new("reset_payload").unwrap());
+// // TODO: this will be deprecated in 5.0.11
```

Comment:
Remove comment, or change to : "has been deprecated"?

Reply:
fixed